tasks/ostree-image-compose

```diff
@@ -10,6 +10,7 @@ base_dir="$(dirname $0)/.."
 HOMEDIR=$(pwd)
 rm -rf ${HOMEDIR}/output
 mkdir -p ${HOMEDIR}/output
+ln -fs ${HOMEDIR}/output ${HOMEDIR}/logs
```

Comment:
weird I thought it ran with your change and failed for the same reason when I went to the node for that stage name logs was not there.